OpenSim/Simulation/Manager/Manager.h

```diff
@@ -173,11 +167,25 @@ class OSIMSIMULATION_API Manager
     { _writeToStorage =  writeToStorage; }
 
     // Integrator
+    enum class Integrator {
+        CPodes,
+        ExplicitEuler,
+        RungeKutta2,
+        RungeKutta3,
+        RungeKuttaFeldberg,
+        RungeKuttaMerson,
+        //SemiExplicitEuler, no error ctrl, requires fixed stepSize arg on construction
+        SemiExplcitEuler2,
+        Verlet
+    };
+    void setIntegrator(Integrator integMethod);
+    void setIntegrator(const std::string& integMethod);
     SimTK::Integrator& getIntegrator() const;
-    /** %Set the integrator. The Manager does *not* take ownership of the
-     * passed-in integrator.
-     */
-    void setIntegrator(SimTK::Integrator&);
+    void setAccuracy(double accuracy);
+    void setMinimumStepSize(double hmin);
+    void setMaximumStepSize(double hmax);
+    void setInternalStepLimit(int nSteps);
```

Comment:
For all of these, perhaps link to the Simbody documentation (e.g., `/** See SimTK::Integrator::setAccuracy(). */`).

If you built and installed Simbody's doxygen before building OpenSim's, the links should go to your local copy of Simbody's doxygen.